### ISSUE TITLE
Links not working in PDF?

### ISSUE THREAD

**Issue by Vuccappella** [Renovamen/oh-my-cv#13]
Upon exporting my CV in PDF with links to my LinkedIn/Portfolio and email (mailto: link)  I noticed that when i open the pdf all the links are not imported and dont work. 

Pretty big issue as I would not send a CV that doesnt have links working and recruiters unfortunately dont accept markdown :D

**Comment by Vuccappella** [comment]
Note: Im specifically talking about the links in the header as they seem to be special and defined in a special way through the editor.

I tried using normal markdown syntax to replace them using [Link text Here](https://link-url-here.org) syntax but that just broke the header. 

I haven't tested links in the body.

**Comment by Renovamen** [comment]
Thanks for pointing that out. Could you let me know which browser you are using? I rarely test on Windows, so I might have missed something...

**Comment by Vuccappella** [comment]
I'm using Vivaldi which is chromium based. Microsoft Print to PDF seems to be like a screnshot, no links are working or text selectable. Standard "PDF" option works as expected.

**Comment by Renovamen** [comment]
Thanks for bringing this issue to my attention. I did some testing and it seems to be a common issue for Chromium browsers on Windows (except Chrome), where the default option is always Microsoft Print to PDF. I searched but couldn't find a way to change the default print option on the app's side. So I'll add a tip to the app and documentation mentioning this.

BTW, it's the first time I've tried the Vivaldi browser and I think I like it. Thanks for mentioning it to me🤣
